### PR TITLE
Fix link in image-layout.md

### PR DESCRIPTION
**Pull request by coolljt0725**
Signed-off-by: Lei Jitang <leijitang@huawei.com>

### CODE REVIEW

File: image-layout.md [hunk 1]
```diff
@@ -40,8 +40,8 @@ afff3924849e458c5ef237db5f89539274d5e609db5db935ed3959c90f1f2d51 ./blobs/sha256/
 
 Object names in the `refs` subdirectories MUST NOT include characters outside of the set of "A" to "Z", "a" to "z", "0" to "9", the hyphen `-`, the dot `.`, and the underscore `_`.
 Object names in the `blobs` subdirectories are composed of a directory for each hash algorithm, the children of which will contain the actual content.
-A blob, referenced with digest `<alg>:<hex>` (per [descriptor][descriptor#digests-and-verification]), MUST have its content stored in a file under `blobs/<alg>/<hex>`.
-The character set of the entry name for `<hex>` and `<alg>` MUST match the respective grammar elements described in [descriptor][descriptor#digests-and-verification]).
+A blob, referenced with digest `<alg>:<hex>` (per [descriptor](descriptors#digests-and-verification)), MUST have its content stored in a file under `blobs/<alg>/<hex>`.
+The character set of the entry name for `<hex>` and `<alg>` MUST match the respective grammar elements described in [descriptor](descriptors#digests-and-verificationi).
```
Comment (runcom) [inline image-layout.md]: typo s/verificationi/verification/

Comment (coolljt0725) [inline image-layout.md]: @runcom  fixed, thx :smile:

**Comment by runcom** [comment]
LGTM (IANAM)

**Comment by philips** [comment]
LGTM 

The doc generator works right with this right @vbatts ?

**Comment by jonboulle** [comment]
Isn't this a link to the embedded link at the bottom of the document? (doesn't that work with subsections?)

**Comment by wking** [comment]
On Thu, Sep 01, 2016 at 06:00:40AM -0700, Jonathan Boulle wrote:
> Isn't this a link to the embedded link at the bottom of the
> document? (doesn't that work with subsections?)

You can define *new* anchors for particular subsections.  E.g.:

  [digest]: descriptor.md#digests-and-verification

But […][digest#…] does not work [1].

[1]: https://github.com/opencontainers/image-spec/blob/08ef0b1d662ac889d3bf2e99957acddad8e5804f/image-layout.md#open-container-initiative-image-layout-specification

**Comment by stevvooe** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/opencontainers/image-spec/pull-request/236/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=opencontainers/image-spec)

**Comment by jonboulle** [comment]
lgtm

**Comment by stevvooe** [comment]
My `(``)` keys must have been broken that day...

**Pull request merged**